docs/synergy/operating-synergy.md

```diff
@@ -16,6 +16,9 @@ the server. Media content will be automatically transcoded to the AiM
 codec as required by the Ai system. This is very handy when you are
 handed a last-minute media file at the front of house position.
 
+As like as any other window the media browser window is opened through the
+\[Open Workspace Window\] menu.
```

Comment:
```suggestion
As with other windows, the media browser window is opened through the
\[Open Workspace Window\] menu, by pressing \<Open\>/\<View\>.
```

Reply:
Suggest using the double press "View" method rather than the Workspace Windows menu as we are recommending that everywhere else in the manual.

Reply:
Would you like me to make this change?

Reply:
yes please...

Reply:
Yes I am not sure how to edit your edit